src/config.ts

```diff
@@ -25,6 +25,10 @@ const INTL_DIR = `${WORKING_DIR}/intl`;
 const projectLanguages: string[] = process.env.CROWDIN_LANGUAGES.split(
   ","
 ) as string[];
+const preTranslatedLanguages: string[] = process.env
+  .CROWDIN_PRE_TRANSLATED_LANGUAGES
+  ? (process.env.CROWDIN_PRE_TRANSLATED_LANGUAGES.split(",") as string[])
+  : projectLanguages;
```

Comment:
This fallback will break the DeepL integration as DeepL doesn’t support `nl-BE` for example. I’ve created a branch in this repo. You can rebase your PR onto that one, then you can use `DEEPL_SUPPORTED_LANGUAGES` instead.